README.md

```diff
@@ -15,6 +15,32 @@ To install this module in Julia, use
 ] add https://github.com/oscar-system/GAP.jl
 ```
 
+## Basic usage
+
+After entering the following in Julia,
+```julia
+using GAP
+```
+one may call any GAP function by prefixing its name with `GAP.Globals.`. For example:
+```julia
+julia> GAP.Globals.SymmetricGroup( 3 )
+GAP: SymmetricGroup( [ 1 .. 3 ] )
+```
+The Julia types `Int64` and `Bool` are automatically converted to GAP objects.
+Several others types of objects can be converted using `GAP.julia_to_gap`:
+```julia
+julia> x = GAP.julia_to_gap([1,2,3])
+GAP: [ 1, 2, 3 ]
+```
+Converting back to Julia is done using `GAP.gap_to_julia`. However, for this one needs to specify the desired type of the resulting object. For example, to convert the GAP list of integers we just defined back to Julia, we might do this:
```

Comment:
Why is this one long line, while lines 29/30 is wrapped?

Reply:
I think because you made this line longer because of your suggestions ;). Anyway, fixed it.